app/components/UpdateNodeForm.jsx

```diff
@@ -2,12 +2,19 @@ import React, { Component, PropTypes } from 'react';
 import BaseComponent from './BaseComponent';
 import { HotKeys } from 'react-hotkeys';
 import merge from 'lodash/object/merge';
+import ChangeColorInput from './ChangeColorInput'
 
 export default class UpdateNodeForm extends BaseComponent {
 
+
   render() {
+
     let { display } = this.props.data;
 
+    if (!display.color){
+      display.color = "#cccccc";
```

Comment:
we should use the default `circleColor` from `NodeDisplaySettings` so the color settings stay in one place